code/modules/surgery/organs/autosurgeon.dm

```diff
@@ -23,6 +23,11 @@
 	name = "[initial(name)] ([storedorgan.name])"
 
 /obj/item/autosurgeon/attack_self(mob/user)//when the object it used...
+	if(istype(storedorgan, /obj/item/organ/regenerative_core))
+		var/obj/item/organ/regenerative_core/cores = storedorgan
```

Comment:
Any reason why this is typed instead of deleting the stored organ

Reply:
I am stupid and this was the quickest way I knew how to make it work